pkg/strategy/output.go

```diff
@@ -0,0 +1,151 @@
+package strategy
+
+import (
+	"fmt"
+	"io"
+	"reflect"
+	"sort"
+	"strings"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+	"github.com/c9s/bbgo/pkg/util"
+	"github.com/fatih/color"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+type JsonStruct struct {
+	Key   string
+	Json  string
+	Type  string
+	Value interface{}
+}
+
+type JsonArr []JsonStruct
+
+func (a JsonArr) Len() int           { return len(a) }
+func (a JsonArr) Less(i, j int) bool { return a[i].Key < a[j].Key }
+func (a JsonArr) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+
+func DefaultStyle() *table.Style {
+	style := table.Style{
+		Name:    "StyleRounded",
+		Box:     table.StyleBoxRounded,
+		Format:  table.FormatOptionsDefault,
+		HTML:    table.DefaultHTMLOptions,
+		Options: table.OptionsDefault,
+		Title:   table.TitleOptionsDefault,
+		Color:   table.ColorOptionsYellowWhiteOnBlack,
+	}
+	style.Color.Row = text.Colors{text.FgHiYellow, text.BgHiBlack}
+	style.Color.RowAlternate = text.Colors{text.FgYellow, text.BgBlack}
+	return &style
+}
+
+func DefaultWhiteList() []string {
+	return []string{"Window", "Interval", "Symbol"}
+}
+
+// @param s: strategy object
+// @param f: io.Writer used for writing the config dump
+// @param style: pretty print table style. Use DefaultStyle() to get default one.
+// @param withColor: whether to print with color
+// @param whiteLists: fields to be printed out from embedded struct (1st layer only)
+func PrintConfig(s interface{}, f io.Writer, style *table.Style, withColor bool, whiteLists ...string) {
```

Comment:
move to pkg/dynamic?

Reply:
there'll be a import cycle if I put this in `pkg/dynamic`.